andela/workdey
# #[123043311] taskee references

#### What does this PR do?

Adds Taskee references feature
#### Description of Task to be completed?

A taskee should be able to add a reference and the added reference should receive and email with instructions of how to endorse the taskee.
#### How should this be manually tested?
- Log in a taskee
- Click `My References`
- Fill form and submit

Reference should receive an email
#### What are the relevant pivotal tracker stories?

123043311
#### Screenshots (if appropriate)
